### PR TITLE
Add PHP tests directory structure section

### PR DESCRIPTION
**Pull request by nunorafaelrocha**
This adds the `Tests/` directory structure standard.

R= @fixe , @ruimarinho , @nunofgs , @franciscocardoso .

### CODE REVIEW

File: php/README.md [hunk 1]
```diff
@@ -147,3 +148,40 @@ use OtherVendor\OtherPackage\Qux;
 
 // ...
 ```
+
+## Tests directory structure
+
+Inside the `Tests` directory, Unit and functional tests *must* be in separated folders - `Functional/` and `Unit/`.
```
Comment (ruimarinho) [inline php/README.md]: separate

Comment (ruimarinho) [inline php/README.md]: First `Unit` should be `unit.

**Comment by ruimarinho** [comment]
LGTM minus the small comments.

**Comment by nunorafaelrocha** [comment]
@ruimarinho Done :)

**Pull request merged**